rst_files/getting_started.rst

```diff
@@ -224,41 +214,25 @@ Now try copying the following into a notebook cell and hit ``Shift-Enter``
     gr(fmt=:png)
     plot(sin, -2π, 2π, label="sin(x)")
 
-You'll see something like this (although the style of plot depends on your
-installation --- more on this later)
 
 **Note**: The "time-to-first-plot" in Julia takes a while, since it needs to precompile everything
 
-.. figure:: /_static/figures/nb4_julia.png
-   :scale: 100%
-
-
-Working with the Notebook
------------------------------
-
-Let's go over some more Jupyter notebook features --- enough so that we can press ahead with programming
-
 
 Tab Completion
 ^^^^^^^^^^^^^^^^^^
 
-A simple but useful feature of `IJulia <https://github.com/JuliaLang/IJulia.jl>`_ 
-(the Jupyter kernel for Julia) is tab completion
+Tab completion in Jupyter makes it easy to find Julia commands and functions available
 
 For example if you type ``rep`` and hit the tab key you'll get a list of all
 commands that start with ``rep``
 
 .. figure:: /_static/figures/nb5_julia.png
-   :scale: 100%
-
-IJulia offers up the possible completions
-
-This helps remind you of what's available and saves a bit of typing
+   :scale: 80%
 
 
 .. _gs_help:
 
-Online Help
+Getting Help
```

Comment:
Doesn't help also work offline? Also I think we could add a screenshot to this section

Reply:
@arnavs could you help with this?

Reply:
Hey, yeah this definitely works offline.

Reply:
Done. I'm also glad you brought this up, because `repmat` has been deprecated in favor of `repeat`, so people would be especially confused...